.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-# run the test or lint command here
+npm run lint
```

Comment:
Add prettier command also

Reply:
Added lintstage package. It performs eslint and prettier in configuration file `.lintstagedrc.json`